src/commands/autocomplete/path.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import AutocompleteBase from '.'
+import path from 'path'
+
+export default class AutocompletePath extends AutocompleteBase {
+  static topic = 'autocomplete'
+  static command = 'commandspath'
+  static description = 'path to autocomplete commands'
+  static hidden = true
+
+  async run () {
```

Comment:
having a base class I think is a good improvement, but I wouldn't name it 'index.js'. I think it should be named `base.js`. Is that going to screw with how it automatically adds all *.js files as commands though?